dashboard/app/models/pd/application/tag.rb

```diff
@@ -0,0 +1,17 @@
+# == Schema Information
+#
+# Table name: pd_application_tags
+#
+#  id   :integer          not null, primary key
+#  name :string(255)      not null
+#
+# Indexes
+#
+#  index_pd_application_tags_on_name  (name)
+#
+
+module Pd::Application
+  class Tag < ActiveRecord::Base
+    self.table_name = 'pd_application_tags'
```

Comment:
Do you need to specify this? I think rails should do this automatically

Reply:
without it I get this error when I try and create a tag in the console: `ActiveRecord::StatementInvalid (Mysql2::Error: Table 'dashboard_development.pd_tags' doesn't exist: SHOW FULL FIELDS FROM 'pd_tags')`